docs/components/modeler/feel/builtin-functions/feel-built-in-functions-list.md

```diff
@@ -80,10 +80,10 @@ min(1, 2, 3);
 - result: number
 
 ```js
-product([2, 3, 4]);
+product([2, 3, 4])
```

Comment:
🔧  Is there a convention about spaces between list-items? In these docs you sometimes do have them and sometimes you don't. It would be nice to stick with 1 of those throughout.

Reply:
You can have whitespace or not. There is no difference.  

I see your point in taking one approach. But I don't see a strong need for it now. There are other parts in the docs that could be improved.